Framework/DataHandling/src/SaveGSS.cpp

```diff
@@ -608,11 +739,6 @@ void SaveGSS::setOtherProperties(IAlgorithm *alg,
   * @throws :: If for any reason we cannot run the algorithm
   */
 void SaveGSS::validateUserInput() const {
-  // Check whether it is PointData or Histogram
```

Comment:
Why did you remove this check? Does the implementation of that have an assumption about constant number of bins?

Reply:
Histogram can give correct result for both histogram and point data.  Therefore, there is no need to check anymore.  The check itself is a legacy.  I tested histogram and point data from same workspace, and got same result in GSAS file.